website/pages/en/index.js

```diff
@@ -85,7 +85,7 @@ class HomeSplash extends React.Component {
               <FaMagic size="3em" />
               <h4>Run From Anywhere</h4>
               <p>
-                Issueist is a PWA, giving it incredible flexibility. Download the chrome extension, or just embed <a>https://issueist.wolfpak.now.sh</a> anywhere in your browser to use it.
+                Issueist is a PWA, giving it incredible flexibility. Download the chrome extension, or just embed <a href="https://issueist.wolfpak.now.sh" rel="noopener noreferrer" target="_blank">https://issueist.wolfpak.now.sh</a> anywhere in your browser to use it.
```

Comment:
https://github.com/kaufmann42/issueist/blob/final-touches/public/manifest.json

The `start_url` in the PWA manifest should probably point to https://issueist.wolfpak.now.sh

Also iirc to *actually* be a PWA, you need to enable a service worker. We need to actually test it out on mobile and see if we get the prompt to download